### PR TITLE
Add a motd for useful BACH variables

### PR DESCRIPTION
**Pull request by http-418**
This PR fixes issue #237.

```
Welcome to Ubuntu 12.04.5 LTS (GNU/Linux 3.2.0-86-virtual x86_64)

You have logged into a member of a BACH cluster.

Environment: Test-Laptop
Bootstrap node IP: 10.0.100.3
Roles:
  - role[BCPC-Bootstrap]
```

### CODE REVIEW

**Comment by http-418** [comment]
Added Zabbix and Graphite URLs to the motd.

```
Welcome to Ubuntu 12.04.5 LTS (GNU/Linux 3.2.0-86-virtual x86_64)

You have logged into a member of a BACH cluster.

Environment: Test-Laptop
Zabbix URL: https://10.0.100.5:7777
Graphite URL: https://10.0.100.5:8888
Bootstrap node IP: 10.0.100.3
Roles:
  - role[BCPC-Bootstrap]
```

File: cookbooks/bach_common/metadata.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name             'bach_common'
+maintainer       'Bloomberg LP'
+maintainer_email 'compute@bloomberg.net'
+license          'All rights reserved'
+description      'bach_common contains common recipes for the BACH cookbooks'
+long_description 'bach_common contains common recipes for the BACH cookbooks'
+version          '0.1.0'
+
+supports 'ubuntu', '12.04'
```
Comment (kiiranh) [inline cookbooks/bach_common/metadata.rb]: DO we want to specify version here? We don't do that in other cookbooks.

File: cookbooks/bach_common/templates/default/motd/bach-variables.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Environment: <%= node.chef_environment %>
+Zabbix URL: https://<%= node[:bcpc][:management][:vip] %>:<%= node[:bcpc][:zabbix][:web_port] %>
+Graphite URL: https://<%= node[:bcpc][:management][:vip] %>:<%= node[:bcpc][:graphite][:web_port] %>
+Bootstrap node IP: <%= node[:bcpc][:bootstrap][:server] %>
+Roles:
+<% node.run_list.each do |entry| %>
+<%= "  - #{entry.to_s}" %>
+<% end %>
+
```
Comment (kiiranh) [inline cookbooks/bach_common/templates/default/motd/bach-variables.erb]: A message specifying if the cluster is secure(kerberized) would be good-to-have.

**Comment by amithkanand** [comment]
Tested this on a dev cluster and looks good.

**Pull request merged**